src/upcoming-components/Accordion/Accordion.tsx

```diff
@@ -4,19 +4,9 @@ import styles from './Accordion.module.css';
 import AccordionPanel from '../AccordionPanel';
 
 export interface Props {
-  /**
-   * Child node(s) that can be nested inside component
-   */
-  children?: ReactNode;
-  /**
-   * CSS class names that can be appended to the component.
-   */
-  className?: string;
-  /**
-   * Inverted variant
-   * 1) Used for dark backgrounds
-   */
-  inverted?: boolean;
```

Comment:
I think leaving prop comments like this is fine, keeps it more readable and allows intellisense and jsdoc to pick up it up

Reply:
+1 I prefer this style